config/webpack.config.js

```diff
@@ -197,6 +275,50 @@ const configGenerator = (buildOptions, apps) => {
         fileName: 'file-manifest.json',
       }),
     );
+  } else {
+    // Add landing pages for local development with the watch task
+    baseConfig.plugins = baseConfig.plugins.concat(
+      getAppManifests()
+        .map(m => {
+          // There's no place to create a landing page at
+          if (!m.rootUrl) return undefined;
```

Comment:
Should this throw an error? I didn't know we had manifests without this.

Reply:
I didn't either. :man_shrugging: But it turns out it's a thing. Rather than figure out why and change folks' applications, I figured I'd just not generate a dev landing page for them unless a `rootUrl` exists.

I don't think it makes sense to break everybody's watch task if some unrelated application's `manifest.json` doesn't specify a `rootUrl`.

Reply:
Actually, `static-pages` doesn't have a `rootUrl`, so that makes sense 😅 Although that bundle is a special case